docs/trigger_moderate.md

```diff
@@ -0,0 +1,105 @@
+# Trigger moderate
+
+Let's say you want to moderate comments using OpenAI.
+
+You can get the full example in the [trigger_moderate.sql](examples/trigger_moderate.sql) file.
+
+First, let's create the extension and set the API key:
+
+```sql
+create extension if not exists ai cascade;
+```
+
+To set the API key, you can use the following command:
+```sql
+select set_config('ai.openai_api_key', :'OPENAI_API_KEY', false) is not null as set_config;
```

Comment:
oops. I set a bad example in the tests.sql. We should use $1 and \bind to help minimize the risk of having the API key printed to logs etc. 

```suggestion
\getenv openai_api_key OPENAI_API_KEY
select set_config('ai.openai_api_key', $1, false) is not null as set_openai_api_key
\bind :openai_api_key
\g
```

Reply:
Fixing tests.sql https://github.com/timescale/pgai/pull/15